lib/luaT/README.md

```diff
@@ -225,11 +225,27 @@ Assuming `tname` is of the form `modulename.classname`, returns
 inside `tname` string.
```

Comment:
Can you also update the docs for `luaT_classrootname` to clarify what happens for nested modules?